### PR TITLE
Slight changes to camera network code

### PR DESCRIPTION
**Pull request by Jalleo**
Removes a useless var and corrects two cameras to use the automatic naming system to reduce direct var usage.

### CODE REVIEW

**Comment by SpaceManiac** [comment]
```
_maps/map_files/BoxStation/BoxStation.dmm:12249:error: invalid variable name: c_tag_order

_maps\map_files\BoxStation\BoxStation.dmm
12249:  c_tag_order = 999;
```

**Comment by Jalleo** [comment]
damn I shall fix this now

File: _maps/shuttles/aux_base_default.dmm [hunk 1]
```diff
@@ -44,10 +44,6 @@
 /obj/effect/turf_decal/stripes/line,
 /turf/open/floor/plating,
 /area/shuttle/auxillary_base)
-"y" = (
```
Comment (Jalleo) [inline _maps/shuttles/aux_base_default.dmm]: not sure what happened here I will check tonight

Comment (SpaceManiac) [reply _maps/shuttles/aux_base_default.dmm]: Probably fine, got mapmerged, judging by the whitespace the last person who touched this used FastDMM but didn't mapmerge it.

**Review by SpaceManiac — APPROVED** [review]

File: _maps/shuttles/aux_base_default.dmm (hunk 1 above)
Comment (ShizCalev) [reply _maps/shuttles/aux_base_default.dmm]: blame goes to @ninjanomnom :v

File: code/game/machinery/camera/tracking.dm [hunk 2]
```diff
@@ -146,10 +146,6 @@
 		for (var/j = 1 to i - 1)
```
Comment (Jalleo) [inline code/game/machinery/camera/tracking.dm]: This /proc/camera_sort should probably be rework to be sorted instead and storing lists like this which need to be ordered as a GLOB for efficency and processing storage but that is what I will do for the next PR. (I have further plans which will probably include doing this)

**Pull request merged**